Version_1.17/src/main/java/simplepets/brainsynder/nms/entity/list/EntityGoatPet.java

```diff
@@ -11,12 +11,10 @@
 import simplepets.brainsynder.nms.entity.EntityAgeablePet;
 
 /**
- * NMS: {@link net.minecraft.world.entity.animal.goat.Goat}
+ * NMS: {@link net.minecraft.server.v1_16_R3.EntityGoat}
```

Comment:
Out of curiosity, how come this is switched to server.v1_16_R3.EntityGoat instead of the MojMap names? This is 1.17, right?